src/core/MidiClockToCV.cpp

```diff
@@ -34,6 +34,20 @@ struct MIDIClockToCVInterface : MidiIO, Module {
 	bool reset = false;
 	int c_bar = 0;
 
+	/* Note this is in relation to the Midi clock's Tick (6x per 16th note).
+	 * Therefore, e.g. the 2:3 is calculated:
+	 *
+	 * 24 (Ticks per quarter note) * 2 / 3 = 16
+	 *
+	 * Implying that every 16 midi clock ticks we need to send a pulse
+	 * */
+	int ratios[] = {6, 8, 12, 16, 24, 32, 48, 96, 192};
```

Comment:
This doesn't build for me (Windows, gcc 7.2.0):

`error: flexible array member 'MIDIClockToCVInterface::ratios' not at end of 'struct MIDIClockToCVInterface'`

Fixing the array size with `ratios[9]` works.